laravel/database/seeds/RoleSeeder.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-use Illuminate\Database\Seeder;
 use App\Models\Role;
+use Illuminate\Database\Seeder;
 
-class CreateRoles extends Seeder
+class RoleSeeder extends Seeder
 {
     private $roles =
-    [
+        [
```

Comment:
It looks like this file was improperly auto-formatted? This whitespace was added and all of the function calls have spaces before the parentheses now.